backend/project_alpha/web/utils/generate_avatar_folder_name.py

```diff
@@ -0,0 +1,3 @@
+def generate_avatar_folder_name(self, filename):
+    path = "users_profile_images/%s/%s" % (self.user.nickname, filename)
```

Comment:
nit: `return f'users_profile_images/{(self.user.nickname}/${filename}'` or something like that. (see "string interpolation in python")